integration_tests/grpc_rest/pki/helpers.go

```diff
@@ -1292,4 +1391,210 @@ func Demo(suite *utils.TestSuite) {
+		Vid:                  vendorAccount.VendorID,
+		IssuerSubjectKeyID:   testconstants.SubjectKeyIDWithoutColons,
+		IsPAA:                true,
+		CrlSignerCertificate: testconstants.PAACertWithNumericVid,
+		Label:                "label",
+		DataURL:              testconstants.DataURL,
+		RevocationType:       1,
+	}
+	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgAddPkiRevDistPoints}, vendorName, vendorAccount)
+	require.NoError(suite.T, err)
+
+	revDistPoints, _ = GetAllPkiRevocationDistributionPoints(suite)
+	require.Equal(suite.T, 1, len(revDistPoints))
+	require.Equal(suite.T, vendorAccount.VendorID, revDistPoints[0].Vid)
+	require.Equal(suite.T, "label", revDistPoints[0].Label)
+	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPoints[0].IssuerSubjectKeyID)
+
+	revDistPoint, _ := GetPkiRevocationDistributionPoint(suite, vendorAccount.VendorID, testconstants.SubjectKeyIDWithoutColons, "label")
+	require.Equal(suite.T, vendorAccount.VendorID, revDistPoint.Vid)
+	require.Equal(suite.T, "label", revDistPoint.Label)
+	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPoint.IssuerSubjectKeyID)
+
+	revDistPointsBySubj, _ := GetPkiRevocationDistributionPointsBySubject(suite, testconstants.SubjectKeyIDWithoutColons)
+	require.Equal(suite.T, vendorAccount.VendorID, revDistPointsBySubj.Points[0].Vid)
+	require.Equal(suite.T, "label", revDistPointsBySubj.Points[0].Label)
+	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPointsBySubj.IssuerSubjectKeyID)
+
+	// Add revocation distribution point for PAI by Vendor
+	//	venName65522 := utils.RandString()
+	//	venAcc65522 := test_dclauth.CreateVendorAccount(
+	//		suite,
+	//		venName65522,
+	//		dclauthtypes.AccountRoles{dclauthtypes.Vendor},
+	//		65522,
+	//		jackName,
+	//		jackAccount,
+	//		aliceName,
+	//		aliceAccount,
+	//		testconstants.Info,
+	//	)
+	//	require.NotNil(suite.T, venAcc65522)
+	//
+	//	// Jack (Trustee) propose Root certificate
+	//	msgProposeAddX509RootCert = pkitypes.MsgProposeAddX509RootCert{
+	//		Cert:   testconstants.PAACertNoVid,
+	//		Signer: jackAccount.Address,
+	//		Vid:    venAcc65522.VendorID,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgProposeAddX509RootCert}, jackName, jackAccount)
+	//	require.NoError(suite.T, err)
+	//	proposedCertificate, err = GetProposedX509RootCert(suite, testconstants.PAACertNoVidSubject, testconstants.PAACertNoVidSubjectKeyID)
+	//
+	//	// Alice (Trustee) approve Root certificate
+	//	secondMsgApproveAddX509RootCert = pkitypes.MsgApproveAddX509RootCert{
+	//		Subject:      proposedCertificate.Subject,
+	//		SubjectKeyId: proposedCertificate.SubjectKeyId,
+	//		Signer:       aliceAccount.Address,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&secondMsgApproveAddX509RootCert}, aliceName, aliceAccount)
+	//	require.NoError(suite.T, err)
+	//
+	//	// Add PAI certificate
+	//	msgAddX509Cert = pkitypes.MsgAddX509Cert{
```

Comment:
There is no need to add a PAI cert to the ledger. It will work even for PAIs not present on the ledger. The only requirements is that PAI needs to be signed (chained back) by a PAA (root) present on the ledger.